Adyen/Model/Amount.swift

```diff
@@ -87,4 +87,10 @@ extension Amount: Comparable {
         lhs.value > rhs.value
     }
 
+    public static func - (lhs: Amount, rhs: Amount) -> Amount {
+        AdyenAssertion.assert(message: "Currency should match to compare",
+                              condition: lhs.currencyCode != rhs.currencyCode)
+        return Amount(value: lhs.value - rhs.value, currencyCode: lhs.currencyCode)
+    }
```

Comment:
I think this is not needed, its best to leave it internal, until its needed

Reply:
But we use it both SDK and in Tests. I see no harm in having it public 🤔

Reply:
in tests we can use @testable, the harm is that it will be public interface, and we'll bound to maintain it.

Reply:
👍